spec/lib/synced/strategies/full_spec.rb

```diff
@@ -6,6 +6,21 @@
   let(:remote_objects) { [remote_object(id: 42, name: "Remote")] }
 
   describe "#perform" do
+    context "with alias" do
```

Comment:
I'm not sure that tests anything. By passing remote you skip api call, no? So it's not testing if it queries proper endpoint.